dd-sdk-android/apiSurface

```diff
@@ -1253,6 +1253,7 @@ abstract class com.datadog.android.rum.tracking.ActivityLifecycleTrackingStrateg
   override fun onActivityStopped(android.app.Activity)
   override fun onActivityCreated(android.app.Activity, android.os.Bundle?)
   override fun onActivityResumed(android.app.Activity)
+  protected fun convertToRumAttributes(android.content.Intent?): Map<String, Any?>
```

Comment:
Shouldn't we also add a monitor and a nightly test for this use case ?

Reply:
No, it'll already be taken care of by the callers (tracking activities etc…). I'm not even sure why it is public.